website/content/docs/secrets/terraform.mdx

```diff
@@ -152,18 +152,78 @@ token           <example token>
 token_id        at-fqvtdTQ5kQWcjUfG
 ```
 
-Please see the [HCP Terraform API
-Token documentation for more
-information](/terraform/cloud-docs/users-teams-organizations/api-tokens).
+### Team roles
 
-## Tutorial
+The HCP Terraform secret engine creates dynamic Team API tokens by 
+configuring a Vault role to manage an existing HCP Terraform Team. 
+The lifecycle of these tokens is managed by Vault and/or HCP Terraform.
+Generally, Vault aims to manage tokens in external APIs with revoke actions
+taken according to ttl & max_ttl settings. In addition to that behavior,
+using the max_ttl Vault will set an `ExpiredAt` date in HCP Terraform 
+also which will ensure the token expires at the Max TTL time. This prevents 
+Team tokens living past their max ttl if Vault is unable to revoke the token.
+Omitting a max_ttl will Vault's system max_ttl
```

Comment:
The last sentence appeared to be incomplete?

```suggestion
Generally, Vault aims to manage tokens in external APIs with revoke actions
taken according to `ttl` and `max_ttl` settings. In addition to that behavior,
using the `max_ttl` Vault will set an `ExpiredAt` date in HCP Terraform 
which will ensure the token expires at the Max TTL time. This prevents 
Team tokens living past their `max_ttl` if Vault is unable to revoke the token.
Omitting the `max_ttl` value will default to Vault's system `max_ttl`.
```

Reply:
roasted 😭 

(jk) appreciate your thorough review :) keep um coming!